view.c

```diff
@@ -258,14 +258,6 @@ void
 view_destroy(struct cg_view *view)
 {
 	struct cg_server *server = view->server;
-	bool ever_been_mapped = true;
-
-#if CAGE_HAS_XWAYLAND
-	if (view->type == CAGE_XWAYLAND_VIEW) {
-		struct cg_xwayland_view *xwayland_view = xwayland_view_from_view(view);
-		ever_been_mapped = xwayland_view->ever_been_mapped;
```

Comment:
`xwayland_view->ever_been_mapped` was only used here. Can you remove its definition in xwayland.h and the setting in xwayland.c as well?